openstack/nova/templates/etc/_nova.conf.tpl

```diff
@@ -203,3 +203,8 @@ default_pool_size = {{ .Values.wsgi_default_pool_size | default .Values.global.w
 
 [workarounds]
 enable_live_migration_to_old_hypervisor = True
+
```

Comment:
An explanatory comment would be helpful here, since these have little-to-no effect and are merely a safeguard (IIUC).

Reply:
I don't think so. If you want to read up on it, look into the commit adding it.